services/login_service/app/certificates.py

```diff
@@ -0,0 +1,20 @@
+import tempfile
+
+
+def create_temp_crt_file(cert_content: str) -> str:
```

Comment:
also nitpick, but name here suggest that you are creating something out of thin air. I think something more like `save_certificate_as_temp_file` will be more readable here